llvm/test/Transforms/LoopVectorize/AArch64/sve-runtime-check-size-based-threshold.ll

```diff
@@ -5,7 +5,6 @@ target triple = "aarch64-unknown-linux-gnu"
 
 ; Test case where the minimum profitable trip count due to runtime checks
 ; exceeds VF.getKnownMinValue() * UF.
-; FIXME: The code currently incorrectly is missing a umax(VF * UF, 28).
```

Comment:
Just noting that the we check for `umax(VF * VF, 20)` due to slightly different cost decisions, but this is still higher than VF.getKnownMinvalue() (2) * UF (2).